otx/cli/tools/optimize.py

```diff
@@ -64,24 +65,14 @@ def parse_args():
     parser.add_argument("--data", required=False, default="./data.yaml")
     required = not os.path.exists("./data.yaml")
 
-    parser.add_argument(
-        "--train-ann-files",
-        required=required,
-        help="Comma-separated paths to training annotation files.",
-    )
     parser.add_argument(
         "--train-data-roots",
         required=required,
         help="Comma-separated paths to training data folders.",
     )
-    parser.add_argument(
-        "--val-ann-files",
-        required=required,
-        help="Comma-separated paths to validation annotation files.",
-    )
     parser.add_argument(
         "--val-data-roots",
-        required=required,
+        required=False,
```

Comment:
Datumaro will do auto-split? Should it be the same as for training to optimize model? I think it should, will it be the same?

Reply:
Yeah, it supports the auto-split and we'll use that function. So, if user doesn't input the `--val-data-roots`, then the auto-split function will be triggered.

Reply:
There is a problem. If user choose auto-split for training how will he optimize his model after that? We need to memorize splitting then

Reply:
Auto-split feature will be considered on Simple Auto-config. So, how about skip this on this PR?

Reply:
> There is a problem. If user choose auto-split for training how will he optimize his model after that? We need to memorize splitting then

I have a question for this, should the trained model be optimized with exactly the same training dataset? 
IMO, 'optimize' does not need to be done with the same training data because it just wants to be optimized with any datasets.